roles/benchmarks/tasks/agent.yml

```
@@ -0,0 +1,98 @@
+---
+- name: Setup kubeconfig
+  set_fact:
+    kubeconfig: "{{archive_dir}}/kubeconfig"
+  when: kubeconfig|length < 1
+
+- name: Check for benchmark-operator
+  shell: |
+    export KUBECONFIG={{kubeconfig}}
+    kubectl get pods -n benchmark-operator --show-labels -l control-plane=controller-manager | grep controller-manager
+  register: bmo
```

There's a check on this return code at the bottom, but why not do a failure condition here first?

---

I think we should move this to a common task file to share for all benchmarks. But agree on failing fast.